src/test/java/com/flightstats/hub/metrics/StatsDFormatterTest.java

```diff
@@ -0,0 +1,67 @@
+package com.flightstats.hub.metrics;
+
+import com.timgroup.statsd.Event;
+import org.junit.Test;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertArrayEquals;
+
+public class StatsDFormatterTest {
+    @Test
+    public void testBuildCustomEvent_Event() {
+        MetricsConfig metricsConfig = MetricsConfig
+                .builder()
+                .hostTag("test_host")
+                .build();
+
+        StatsDFormatter statsDFormatter = new StatsDFormatter(metricsConfig);
+        Event event = statsDFormatter.buildCustomEvent("testTitle", "testText");
+
+        assertEquals("testTitle", event.getTitle());
+        assertEquals("testText", event.getText());
+        assertEquals(Event.Priority.NORMAL.toString().toLowerCase(), event.getPriority());
+        assertEquals(Event.AlertType.WARNING.toString().toLowerCase(), event.getAlertType());
+        assertEquals("test_host", event.getHostname());
+    }
+
+    @Test
+    public void testBuildCustomEvent_Error() {
+        MetricsConfig metricsConfig = MetricsConfig
+                .builder()
+                .build();
+        StatsDFormatter statsDFormatter = new StatsDFormatter(metricsConfig);
+        try {
+            statsDFormatter.buildCustomEvent("", "");
```

Comment:
I think there might be an `assertThrows` or some other assertion that'll handle testing that errors get thrown. This will silently pass if no exception is thrown.

Reply:
good catch, the second of these was silently failing... derp